README.md

```diff
@@ -24,6 +35,12 @@ junit compatible XML to standard out.
 go test -v 2>&1 | go-junit-report > report.xml
 ```
 
+Note that it also can parse benchmark output with `-bench` flag:
+```bash
+ go test -bench . -benchmem -count 100
```

Comment:
Also, I recently found interesting benchmark option:

-benchtime t
    Run enough iterations of each benchmark to take t, specified
    as a time.Duration (for example, -benchtime 1h30s).
    The default is 1 second (1s).

I assume that it works fine with parser but it can be useful to provide in example as well.